### PR TITLE
[PR] Add agent synchronization statistics

### PR DESCRIPTION
**Pull request by yenienserrano**
**Description:**

Added agent synchronization status in the agent view.
And the endpoint response type has been modified `GET /agents/summary/status` so the responses of the views using that endpoint have been fixed.

**Issue:**

- #1309

**Test:**

1. Navigate to the following views:
         - Overview
         - Management/Status 
         - Agents
2. The number of agents in the different states should be displayed. 
3. in the agent module in the details part there should appear Agents synced and in the table, there should be a Synchronized column.

Screenshot:

Overview:

image

Management/Status:

image

Agents:

image

### CODE REVIEW

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agentsCtrl.js [hunk 1]
```diff
@@ -70,7 +70,8 @@ define([
       this.setBrowserOffset = $dateDiffService.setBrowserOffset
       try {
         const parsedResult = agentData.data.data
-        let summary = this.formatAgentStatusData(parsedResult.agent_status)
+        let summary = this.formatAgentStatusData(parsedResult.agent_status.connection)
```
Comment (asteriscos) [inline SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agentsCtrl.js:73]: Many of these variables are not reassigned and therefore can be constants.

**Review by asteriscos — APPROVED** [review]
CR: Use constants for variables that are not reassigned.
Test: :heavy_check_mark: 

image

**Review by chantal-kelm — APPROVED** [review]
CR: :heavy_check_mark:
Test: :heavy_check_mark: 

Screenshot from 2022-07-14 11-23-36
Screenshot from 2022-07-14 11-23-53
Screenshot from 2022-07-14 11-24-06

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
 All notable changes to the Wazuh App for Splunk project will be documented in this file.
+## Wazuh v4.4.0 - Splunk Enterprise v8.1.4, v8.2.2, v8.2.4 - Revision 4401
+
+## Added
+- Added agent synchronization status in the agent module. [#1355](https://github.com/wazuh/wazuh-splunk/pull/1355)
+
+## Fixed
+
+## Changed
+- endpoint `/agents/summary/status` response was adapted. [#1355](https://github.com/wazuh/wazuh-splunk/pull/1355)
```
Comment (AlexRuiz7) [inline CHANGELOG.md:12]: Change to: 
> Update response handlers for the endpoint `/agents/summary/status`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,19 @@
 # Change Log
 
 All notable changes to the Wazuh App for Splunk project will be documented in this file.
+## Wazuh v4.4.0 - Splunk Enterprise v8.1.4, v8.2.2, v8.2.4 - Revision 4401
```
Comment (AlexRuiz7) [inline CHANGELOG.md:4]: Splunk Enterprise v8.1.[1-10], v8.2.x

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agents.html [hunk 4]
```diff
@@ -103,6 +103,17 @@
                 </p>
               </div>
             </div>
+            <div class="euiFlexItem euiFlexItem--flexGrowZero">
+              <div class="euiStat">
+                <div class="euiText euiText--small euiStat__description">
+                  <p>Agents synced</p>
```
Comment (AlexRuiz7) [inline SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agents.html:109]: Change to "Synced agents", as done in [Kibana](https://github.com/wazuh/wazuh-kibana-app/pull/3874/files#diff-9f89d90556bc520d79685e9c92d6d0452a844a9af08c5aba988bd291a6f69462R226)

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

**Review by AlexRuiz7 — APPROVED** [review]
- Code review: ✔️
- Test: ✔️

**Pull request merged**